package.json

```diff
@@ -28,7 +28,7 @@
     }
   },
   "peerDependencies": {
-    "webpack": ">=0.9 <2 || ^2.1.0-beta || ^2.2.0"
+    "webpack": ">=0.9 <2 || ^2.1.0-beta || >=2.2.0 <4"
```

Comment:
@danez - Yes, we can land a quick patch. 

@fb55 - Set the peerDependencies to  `"webpack": ">=0.9 || ^2.0.0 || ^3.0.0"`